### ISSUE TITLE
Mobile SDK Data Privacy Disclaimer

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/ouds-ios#1015]
# Requirement
As a user of the sdk
I need to understand what are the data treatments of the SDK
So that I can report to data privacy authorities about all sdk used in my mobile app.

# Context
The CNiL has issues a recommendation that describes obligations for mobile sdk providers:
 https://www.cnil.fr/sites/cnil/files/2025-04/recommandation-applications-mobiles-modifiee.pdf

In this documentation the objective is to understand the positionning of an sdk relative to the data privacy, and the position of the sdk provider relative to the data manipulated by the sdk.

In the case of the OUDS sdk we do not treat personal data, but we do need to provide the information to the sdk users.

# implementation

## Readme Description
 In the readme file there should be a new paragraph

> ### Data Privacy
> The Orange Unified Design System library is an sdk that allows a developper to create Orange branded mobile application. As such: 
> * this sdk does not handle any personnal data.
> * this sdk does not require any device permission to work. 

## Dev Documentation
Inside the home page of the documentation this same paragraph should be included within the page

**Comment by B3nz01d** [comment]
Related issues for other platforms:

- [ ] Android: # [830](https://github.com/Orange-OpenSource/ouds-android/issues/830)
- [ ] iOS: # [1015](https://github.com/Orange-OpenSource/ouds-ios/issues/1015)
- [ ] Flutter: # [410](https://github.com/Orange-OpenSource/ouds-flutter/issues/410)
